CMakeLists.txt

```diff
@@ -8,7 +8,12 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
-
+set(CMAKE_BUILD_TYPE Debug)
+find_package(Yosys)
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/test/gate/parserGnet/normal1.lib DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/test)
```

Comment:
Do not copy test files: there are still accessible, see the notes on the `normal.lib` file and access to it in the test.
Long story short: remove the line.